scripts/generate_base_coverage.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+import argparse
+import glob
+import json
+import magic
```

Comment:
Is `magic` a system library? I cannot find it in package.xml

Reply:
magic is installed in `gitai_linux_setup` but not found in `ros/rosdistro`.
I will create a rule.
https://github.com/GITAI/gitai_linux_setup/blob/6a7fb03d0a64601b9b097628aac188e47835b85d/playbooks/roles/common-apt/tasks/main.yml#L59

Reply:
doing https://github.com/ros/rosdistro/pull/37973

Reply:
LGTM

Reply:
If it is installed by gitai_linux_setup, it is fine. As you are doing, it is better to send a PR to rosdistro.

Reply:
https://github.com/ros/rosdistro/pull/37973 is merged, so we can install through rosdep.